src/app/rpm-ostree-countme.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Weekly rpm-ostree Count Me reporting
+Documentation=man:rpm-ostree-countme(1)
+ConditionPathExists=/run/ostree-booted
+
+[Service]
+Type=simple
+User=rpm-ostree
+DynamicUser=yes
+StateDirectory=rpm-ostree
```

Comment:
One confusing thing is that we do also use `/var/lib/rpm-ostree` today. Maybe let's call this (and the user) `rpm-ostree-countme`?